nuxeo-ai-config/src/main/java/org/nuxeo/ecm/restapi/server/jaxrs/AIRoot.java

```diff
@@ -166,13 +165,12 @@ public Response getAllThresholds() {
     @Path("extension/model")
     @Consumes(MediaType.APPLICATION_XML)
     @Produces(MediaType.APPLICATION_JSON)
-    public Response setModelFromXML(String modelXML) {
+    public Response setModelFromXML(@QueryParam("modelId") String modelId, String modelXML) {
```

Comment:
why not a `PathParam`? Or at least there should be a null check

Reply:
- I have no opinion on the query param vs path param but I can change it for all endpoints
- Thanks for the review, i will check null